css/components/grid.css

```diff
@@ -0,0 +1,23 @@
+.grid {
+  display: grid;
+  width: 30rem;
+  height: 5rem;
+  align-items: center;
+  gap: 1rem;
+  border: 2px solid var(--primary--color);
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.grid-2 {
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid-3 {
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
You can also write it as "grid-template-columns: repeat(3, 1fr); instead of writing it 3 times